tests/ui/README.md

```diff
@@ -17,19 +17,15 @@ Follow the instructions found [here][addons-server-docs].
 
 ### Run the tests
 
-*IMPORTANT* : Run the script in ```scripts/ui-test-hostname-setup.sh``` before running the test to setup the hostnames within the docker container.
-
-The selenium tests are included in a separate ``tests/ui/docker-compose.selenium.yml`` file. This image contains Firefox Nightly. [tox][Tox]
+The selenium tests can be run in a separate docker-compose file, specifically ``tests/ui/docker-compose.selenium.yml``. This image contains Firefox Nightly. [tox][Tox]
 is our test environment manager and [pytest][pytest] is the test runner.
 
 To run the tests, execute the commands below:
 ```sh
 # Make sure all required containers are running.
 docker-compose -f docker-compose.yml -f tests/ui/docker-compose.selenium.yml up -d
-docker-compose -f docker-compose.yml -f tests/ui/docker-compose.selenium.yml exec --user root selenium-firefox tox -e ui-tests
+docker-compose -f docker-compose.yml -f tests/ui/docker-compose.selenium.yml exec selenium-firefox tox -e ui-tests
 ```
-WARNING: This will WIPE the database as the test will create specific data for itself to look for.
-If you have anything you don't want to be deleted, please do not run these tests.
```

Comment:
it doesn't wipe the database any more?  (last time I tried it, it did!)

Reply:
It does but not that command. I'll move that under the ```make initialize_docker``` command.